unittest.d

```diff
@@ -0,0 +1,5 @@
+
+int main() 
+{
+    return 0;
+}
```

Comment:
This file shouldn't be necessary. You can add `-main` when compiling to let the compiler generate a main function.